sdk/python/kfp/components/executor.py

```diff
@@ -129,9 +136,41 @@ def _write_output_artifact_payload(self, name: str, value: Any):
     with open(path, 'w') as f:
       f.write(str(value))
 
+  # TODO: extract to a util
+  @classmethod
+  def _get_short_type_name(cls, type_name: str) -> str:
+    """Extracts the short form type name.
+
+    This method is used for looking up serializer for a given type.
+
+    For example:
+      typing.List -> List
+      typing.List[int] -> List
+      typing.Dict[str, str] -> Dict
+      List -> List
+      str -> str
+
+    Args:
+      type_name: The original type name.
+
+    Returns:
+      The short form type name or the original name if pattern doesn't match.
+    """
+    import re
+    match = re.match('(typing\.)?(?P<type>\w+)(?:\[.+\])?', type_name)
+    if match:
+        return match.group('type')
```

Comment:
indent 2 spaces to keep it consistent within this file.

Reply:
Done.